src/gordon_gcp/plugins/service/enricher.py

```diff
@@ -222,16 +216,10 @@ def _get_internal_ip(self, instance_data):
         instance_name = instance_resource_url.split('/')[-1]
         fqdn = self._get_fqdn(instance_name)
 
-        # Get the Google DNS A records for managed zone with name matching the
-        # instance fqdn
-        project = self.config['project']
-        managed_zone = self.config['managed_zone']
-        resource_records_url = GCEEnricher.RESOURCE_RECORDS_ENDPOINT.format(
-            project=project, managedZone=managed_zone)
-
+        # Get the A records with name matching the instance FQDN
```

Comment:
FWIW, 'records in the managed zone with' isn't redundant; for private zones you can have the same name/type in two zones.  (I think?)

Reply:
I can't see why you couldn't, but I don't know if that's distinction necessary to make here, since that would change architecture assumptions in a lot of places in gordon.

Reply:
I did bear it in mind in places, but it's probably an acceptable simplification for most use cases.